plugins/lookup/bitwarden.py

```diff
@@ -135,8 +135,8 @@ def run(self, terms, variables=None, **kwargs):
         self.set_options(var_options=variables, direct=kwargs)
         field = self.get_option('field')
         search_field = self.get_option('search')
-        if not _bitwarden.logged_in:
-            raise AnsibleError("Not logged into Bitwarden. Run 'bw login'.")
+        if not _bitwarden.unlocked:
+            raise AnsibleError("Bitwarden Vault locked. Run 'bw unlock'.")
```

Comment:
Does `bw unlock` also run `bw login` when necessary? If not, it should probably be mentioned as well.

Reply:
`bw unlock` does not run it automatically. But it tells the user to login first.


```bash
$ bw unlock
You are not logged in.
```

Reply:
Ah, that should be good enough. Thanks!